_blog/2022-10-08-2.7 release.md

```diff
@@ -0,0 +1,52 @@
+---
+title: A new way to catch up on the latest episodes
+excerpt: Version 2.7, released today, introduces the Inbox!
+date: "2022-10-27 09:00:00"
+image: "2022/version2-7_new.jpg"
```

Comment:
Hmm, not sure about that picture. It feels too generic. For the previous versions, we had a "release banner" thing that we could also post on Twitter. (Template in branding repo)

Reply:
Yes, I explicitly chose not to go for the blue images any-more. (See 2.5 blog post.) They contain text which conflicts with (distracts from) the blog headings, and the whole page becomes boring blue.
I'll see if I can find a more appropriate image.